src/app/containers/Brand/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
-import { bool, node } from 'prop-types';
+import { bool, node, oneOfType, func, shape, any } from 'prop-types';
 import { ServiceContext } from '#contexts/ServiceContext';
 
-const BrandContainer = ({ skipLink, scriptLink, ...props }) => {
+const BrandContainer = ({ skipLink, scriptLink, focusRef, ...props }) => {
```

Comment:
My brain is struggling to remember what this reference is referencing throughout the code. Please can we name **all** references according to the element they are referencing? This will also help future proof this code – we might want to call methods other than `focus` on this element or others down the line.

Reply:
What would you suggest naming this, `linkRef` as it focusses the link in the brand? I personally like the name including focus as it makes clear the intent for the reference's usage.

Reply:
In my opinion I think `linkRef` works well here, now that I think about it the name `focusRef` is maybe a bit odd here, because the focus logic using this reference is actually done in ConsentBanner/index.canonical.jsx.